vscode/src/telemetry.ts

```diff
@@ -0,0 +1,56 @@
+import * as vscode from "vscode";
+import TelemetryReporter from "@vscode/extension-telemetry";
+import { log } from "qsharp";
+// the application insights key (also known as instrumentation key)
+const key = "175861b7-3a41-4015-9571-1d930b8b0722";
```

Comment:
I think the common pattern is to put the key in the package.json and fetch it from there, rather than embed it into the code.